### PR TITLE
Added initial dark theme

### PR DESCRIPTION
**Pull request by gotwig**

### CODE REVIEW

**Comment by gotwig** [comment]
Dont push yet :D I still have to fix some things. Can you maybe work on a theme switcher?

**Comment by julien-moreau** [comment]
Sure !!
I propose to keep the light one by default for instance and switch to the black one once all features have been tested. It includes GUI features available in GUI but not yet used by plugins

**Comment by julien-moreau** [comment]
I’m working on a theme switcher right now :)

**Comment by gotwig** [comment]
Anything you dont like :) ?

File: index-debug.html [hunk 1]
```diff
@@ -10,9 +10,11 @@
     <script src="./node_modules/jquery/dist/jquery.js" type="text/javascript"></script>
     <script src="./node_modules/w2ui/w2ui.js" type="text/javascript"></script>
 
-    <link rel="stylesheet" type="text/css" href="./node_modules/w2ui/w2ui.css" />
```
Comment (julien-moreau) [inline index-debug.html:13]: Restore

File: src/editor/gui/resizable-layout.ts [hunk 2]
```diff
@@ -169,7 +169,7 @@ export default class ResizableLayout {
         this.element.init();
 
         System.import('./node_modules/golden-layout/src/css/goldenlayout-base.css');
-        System.import('./node_modules/golden-layout/src/css/goldenlayout-light-theme.css');
```
Comment (julien-moreau) [inline src/editor/gui/resizable-layout.ts]: Restore

File: index-debug.html [hunk 3]
```diff
@@ -10,9 +10,11 @@
...
+	<!-- <link rel="stylesheet" type="text/css" href="./node_modules/w2ui/w2ui.css" /> -->
     <link rel="stylesheet" type="text/css" href="./node_modules/jstree/dist/themes/default/style.min.css" />
     <link rel="stylesheet" type="text/css" href="./css/editor.css" />
+	<link rel="stylesheet" type="text/css" href="./css/dark.css" />
```
Comment (julien-moreau) [inline index-debug.html]: Remove

File: index-debug.html [hunk 4]
```diff
@@ -10,9 +10,10 @@
     <script src="./node_modules/jquery/dist/jquery.js" type="text/javascript"></script>
     <script src="./node_modules/w2ui/w2ui.js" type="text/javascript"></script>
 
-    <link rel="stylesheet" type="text/css" href="./node_modules/w2ui/w2ui.css" />
+	<link rel="stylesheet" type="text/css" href="./node_modules/w2ui/w2ui.css />
```
Comment (julien-moreau) [inline index-debug.html]: Missing end quote

**Pull request merged**